node-sdk-demo/test/workspaceApi.test.js

```diff
@@ -0,0 +1,120 @@
+import {
+    ApiClient,
+    CreateWorkspaceRequest,
+    EnvOp,
+    ForkEndpoint,
+    ForkOf,
+    Image,
+    WorkspaceCustomizations,
+    WorkspaceFork,
+    WorkspacesApi
+} from '@signadot/signadot-sdk';
+import axios from 'axios';
+import {customAlphabet} from 'nanoid';
+import {expect} from 'chai';
+
+const nanoid = customAlphabet('1234567890abcdef', 5);
+
+let previewURL;
+const SIGNADOT_ORG = 'signadot'; // Enter your Signadot org name here
+const SIGNADOT_API_KEY = process.env.SIGNADOT_API_KEY; // passed from command line
+
+describe('Test Signadot SDK', () => {
+    let workspacesApi, workspaceID;
+    before(async () => {
+        return new Promise(async (resolve, reject) => {
+            try {
+                const apiClient = new ApiClient();
+                apiClient.authentications.ApiKeyAuth.apiKey = SIGNADOT_API_KEY;
+                workspacesApi = new WorkspacesApi(apiClient);
+
+                const request = CreateWorkspaceRequest.constructFromObject({
+                    name: `test-ws-${nanoid()}`,
+                    description: 'created using @signadot/signadot-sdk',
+                    cluster: 'demo',
+                    namespace: 'hotrod',
+                    forks: [
+                        WorkspaceFork.constructFromObject({
+                            forkOf: ForkOf.constructFromObject({
+                                kind: 'Deployment',
+                                name: 'route',
+                                namespace: 'hotrod'
+                            }),
+                            customizations: WorkspaceCustomizations.constructFromObject({
+                                images: [
+                                    Image.constructFromObject({
+                                        image: 'signadot/hotrod-route:540fadfd2fe619e20b794d56ce404761ce2b45a3'
+                                    })
+                                ],
+                                env: [
+                                    EnvOp.constructFromObject({
+                                        name: 'abc',
+                                        value: 'def'
+                                    })
+                                ]
+                            }),
+                            endpoints: [
+                                ForkEndpoint.constructFromObject({
+                                    name: 'hotrod-route',
+                                    port: 8083,
+                                    protocol: 'http'
+                                })
+                            ]
+                        })
+                    ]
+                });
+
+                console.log("Making createNewWorkspace() call.\nRequest:");
+                console.log(JSON.stringify(request, null, 2));
+                const response = await workspacesApi.createNewWorkspace(SIGNADOT_ORG, request);
+                console.log(`Response:\n${JSON.stringify(response, null, 2)}`);
```

Comment:
do these console.log statements need to be removed?

Reply:
It was helpful at times during development. So we can choose to keep it or remove it. Do you suggest removing them?